examples/purchaseTesterTypescript/metro.config.js

```diff
@@ -5,13 +5,36 @@
  * @format
  */
 
-module.exports = {
-  transformer: {
-    getTransformOptions: async () => ({
-      transform: {
-        experimentalImportSupport: false,
-        inlineRequires: true,
-      },
-    }),
-  },
-};
+// IMPORTANT
```

Comment:
do you mind updating our notion docs [here](https://www.notion.so/revenuecat/Setting-the-sample-app-for-development-788b88fabc6f45ba944fe809b1fdb656) once this is merged?